services/brig/test/integration/Util.hs

```diff
@@ -592,7 +592,7 @@ putConnectionQualified brig from (Qualified to toDomain) r =
   where
     payload = RequestBodyLBS . encode $ object ["status" .= r]
 
-connectUsers :: Brig -> UserId -> List1 UserId -> (MonadIO m, MonadHttp m) => m ()
+connectUsers :: (MonadIO m, MonadHttp m) => Brig -> UserId -> List1 UserId -> m ()
```

Comment:
why?

Reply:
My understanding of it: The original version relied on GHC + DeepSubsumption to rewrite this to put the constraint "in the right place". The extension removes the requirement that `forall` must be always in the same place, through eta-expansions and other rewrite rules.